app/src/main/java/uk/org/openseizuredetector/MainActivity2.java

```diff
@@ -105,6 +105,9 @@ protected void onStart() {
         createMainActivity(null);
         setmFragmentPager();
         mFragmentPager.setId(SP.getInt(Constants.GLOBAL_CONSTANTS.lastPagerId,0));
+        serverStatusHandler.postDelayed(()-> {
```

Comment:
This is a funny piece of code - it would be good to add a comment explaining why we need the 400ms delay, just in case I go back in the future and tidy it up and remove the delay (it is not obvious to me why we need the delay).